pineappl_py/src/subgrid.rs

```diff
@@ -195,26 +195,27 @@ impl PySubgridEnum {
         self.subgrid_enum.scale(factor);
     }
 
+    /// Return the dense array of the subgrid.
     pub fn dense<'py>(&self, py: Python<'py>) -> &'py PyArray3<f64> {
         self.subgrid_enum.dense().into_pyarray(py)
     }
 
     pub fn into(&self) -> Self {
         self.clone()
     }
-
+    /// Return the array of mu2 objects of a subgrid
```

Comment:
When you start with "Return" consider replacing with something more informative (all functions return something, or at least they should).

In this case, we could consider just dropping it, and turn the method into a more Pythonic `@property` (PyO3 expose supports this as `#[getter]`).
https://pyo3.rs/v0.18.2/class.html#object-properties-using-getter-and-setter

Reply:
I think this is OK for the time being.